rcl_logging_spdlog/src/rcl_logging_spdlog.cpp

```diff
@@ -152,7 +155,6 @@ rcl_logging_ret_t rcl_logging_external_initialize(
 rcl_logging_ret_t rcl_logging_external_shutdown()
 {
   g_root_logger = nullptr;
```

Comment:
Did you test (with e.g. valgrind) to make sure that the logger is getting cleaned up properly after this patch?  I'm assuming it is, I'm just curious to see the results.

Reply:
I've tested now, it doesn't leak.
I've also double checked the testing procedure with an intentionally leaking version.

Reply:
Fantastic, thanks!